packages/actor-query-operation-orderby-sparqlee/lib/ActorQueryOperationOrderBySparqlee.ts

```diff
@@ -69,7 +70,11 @@ export class ActorQueryOperationOrderBySparqlee extends ActorQueryOperationTyped
         if (!orderA || !orderB) {
           return 0;
         }
-        return orderA > orderB === isAscending ? 1 : -1;
+        if(isAscending){
```

Comment:
Is this change really needed?
If I understand correctly, this defines an order when the two terms are equal.
But if they are equal, the return value should be `0`, right?